### PR TITLE
Restyle home view, some more touchups

### LINKED ISSUE #754: Row overflows with large text on event list

### PR DESCRIPTION
**Pull request by felixrindt**
Fixes #754

### CODE REVIEW

File: ephios/core/templates/core/disposition/disposition.html [hunk 1]
```diff
@@ -14,8 +14,11 @@
 {% endblock %}
 
 {% block content %}
-    <div class="page-header">
-        <h1>{{ shift }}</h1>
+    <div class="page-header text-break">
+        <h1>{{ shift.event }}</h1>
+        <h3 class="subtitle text-muted">
+            {{ shift.get_datetime_display }}
+        </h3>
```
Comment (jeriox) [inline ephios/core/templates/core/disposition/disposition.html:21]: grafik
I think I'd like some more padding on the bottom here

File: ephios/core/templates/core/home.html [hunk 2]
```diff
@@ -51,9 +79,24 @@ <h5 class="card-title mb-0">{% trans "Events with requested participations" %}</
                 <ul class="list-group list-group-flush">
                     {% for shift in shifts %}
                         <li class="list-group-item flex-column">
-                            <div class="d-flex w-100 justify-content-between align-items-center">
-                                <span class="">{{ shift.event.title }} ({{ shift.get_datetime_display }})</span>
-                                <div>
+                            <div class="row">
+                                <div class="col-auto px-0">
+                                    <span class="text-info fa fa-hand-point-right ps-2"></span>
```
Comment (jeriox) [inline ephios/core/templates/core/home.html:84]: grafik
to me it looks like the icon is not perfectly centered?

**Review by jeriox — APPROVED** [review]

File: ephios/core/templates/core/home.html (hunk 2 above)
Comment (felixrindt) [reply ephios/core/templates/core/home.html:84]: Which direction? Vertical or horizontal?

Comment (felixrindt) [reply ephios/core/templates/core/home.html:84]: I think it is right, the icon's finger is just not centered

**Pull request merged**